### ISSUE TITLE
French Translation

### ISSUE THREAD

**Issue by dixours** [ffMathy/Shapeshifter#49]
Hi !

I like Shapeshifter and I would like to give a hand.
May I help you translating this wonderful app into french ?

Please tell me.

Have a good day !

**Comment by ffMathy** [comment]
Yes, it will be possible to contribute with translations of Shapeshifter in the future. However, not right now. When we get out of beta, this will be one of our top priorities.

**Comment by dixours** [comment]
OK !
See you soon !

**Comment by ffMathy** [comment]
@KoalaBear84 any idea of how translation could work by volunteers without paying anything for it? Should we require people to push strings (XML files or so) into GitHub?

**Comment by KoalaBear84** [comment]
We could try first try:
https://www.transifex.com/customers/open-source/
If that doesn't work well:
https://crowdin.com/page/open-source-project-setup-request

**Comment by ffMathy** [comment]
Nice. Crowdin was what Shapeshifter 5 used, but I didn't know they allow for free open source usage.

I contacted them. Thanks once again Koala!

**Comment by ffMathy** [comment]
They already approved us. Awesome! Now we have a translation system we can use for the next Shapeshifter as well!

I will leave this issue open until we actually have a system in place which talks with their API.

**Comment by dixours** [comment]
Nice ! I'm waiting your order (french translation) !! Ah ah !

**Comment by ffMathy** [comment]
Shapeshifter has started development again, but I will postpone this feature until we reach our Patreon goal of $100.
